### ISSUE TITLE
v5: opt-out focus control

### ISSUE THREAD

**Issue by tamb** [twbs/bootstrap#30165]
Part of what is required for accessibility is properly attaching focus to elements (tabs should then focus on tab content, modals on modal body, etc).  This should exist by default but I think it should be opt-out via a data-attribute.

`data-bs-no-focus="true"` and the component initialization will avoid shifting focus.

**Comment by ffoodd** [comment]
Why on earth would someone break components' accessibility on purpose?

**Comment by tamb** [comment]
@ffoodd You may want to control focus to go elsewhere.
